tests/load/test_read_interfaces.py

```diff
@@ -212,6 +212,47 @@ def double_items():
     loads_table = pipeline._dataset()[pipeline.default_schema.loads_table_name]
     loads_table.fetchall()
 
+    # check dataset factory
```

Comment:
we need proper tests to ensure that always the newest schema actually is selected, this is just basic test code to make sure it generally works